migrations/src/main/scala/za/co/absa/enceladus/migrations/framework/migration/CollectionMigration.scala

```diff
@@ -114,12 +114,12 @@ trait CollectionMigration extends Migration {
     * @param collectionName A collection for setting up an index
     * @param fields         A list of fields that the index should contain
     */
-  def createIndex(collectionName: String, fields: Seq[IndexField]): Unit = {
+  def createIndex(collectionName: String, fields: Seq[IndexField], unique: Boolean = false): Unit = {
```

Comment:
Please, update the documentation for this change. Both
- for this function
- the one at the top of the file with an example usage.